ably/realtime/connection.py

```diff
@@ -45,41 +45,136 @@ class ProtocolMessageAction(IntEnum):
 
 
 class Connection(AsyncIOEventEmitter):
+    """Ably Realtime Connection
+
+    Enables the management of a connection to Ably
+
+    Attributes
+    ----------
+    realtime: any
+        Realtime client
+    state: str
+        Connection state
+    connection_manager: ConnectionManager
+        Connection manager
+
+
+    Methods
+    -------
+    connect()
+        Establishes a realtime connection
+    close()
+        Closes a realtime connection
+    ping()
+        Pings a realtime connection
+    on_state_update(state_change)
+        Update and emit current state
+    """
+
     def __init__(self, realtime):
+        """Constructs a Connection object.
+
+        Parameters
+        ----------
+        realtime: any
+            Ably realtime client
+        """
         self.__realtime = realtime
         self.__state = ConnectionState.CONNECTING if realtime.options.auto_connect else ConnectionState.INITIALIZED
         self.__connection_manager = ConnectionManager(realtime, self.state)
         self.__connection_manager.on('connectionstate', self.on_state_update)
         super().__init__()
 
     async def connect(self):
+        """Establishes a realtime connection.
+
+        Causes the connection to open, entering the connecting state
+        """
         await self.__connection_manager.connect()
 
     async def close(self):
+        """Causes the connection to close, entering the closing state.
+
+        Once closed, the library will not attempt to re-establish the
+        connection without an explicit call to connect()
+        """
         await self.__connection_manager.close()
 
     async def ping(self):
+        """
+        Send a ping to the realtime connection
+        """
         return await self.__connection_manager.ping()
 
     def on_state_update(self, state_change):
+        """Update and emit the connection state
+        """
         self.__state = state_change.current
         self.__realtime.options.loop.call_soon(functools.partial(self.emit, state_change.current, state_change))
 
     @property
     def state(self):
+        """Returns connection state"""
```

Comment:
This doesn't match the description from https://github.com/ably/sdk-api-reference/blob/main/descriptions.md, can you make sure these are all matching?